docs/components/table.md

```diff
@@ -257,6 +264,111 @@ export default {
 </script>
 ```
 
+## Custom Row Attributes
+
+There is a prop called `hasSideBorder` that when passed in, enables a default side border on each row, using the color `steal-300` by default.
+This prop is on by default.
+These colors can be overridden by themes or by using custom row attributes.
+Example with `hasSideBorder` set to false:
+
+<template>
+  <KTable
+    :options="tableOptions"
+    :hasSideBorder="false"
+    />
+</template>
+
+```vue
+<template>
+  <KTable
+    :options="tableOptions"
+    :hasSideBorder="false"
+    />
+</template>
+```
+
+We can also add custom row attributes via a prop called `rowAttrs` - this prop is a Function that returns an object comprising the conditional custom attributes
```

Comment:
Lets move this description up (below "Custom Row Attrs") and update the description to not be a continuation.

```suggestion
Add custom properties to individual rows. The row object is passed as a param.

- `rowAttrs` - returns an object comprising the attributes
```